test/integration/framework/controller.go

```diff
@@ -54,7 +54,7 @@ func NewServiceDNSControllerFixture(tl common.TestLogger, config *restclient.Con
 	f := &ControllerFixture{
 		stopChan: make(chan struct{}),
 	}
-	err := servicedns.StartController(config, fedNamespace, clusterNamespace, targetNamespace, f.stopChan, true)
+	err := servicedns.StartController(config, fedNamespace, clusterNamespace, targetNamespace, f.stopChan, 20, 60, true)
```

Comment:
These and all other test-invoked controllers should be using the default constants rather than magic numbers.